examples/moodbot/domain.yml

```diff
@@ -7,7 +7,15 @@ intents:
   - mood_unhappy
   - bot_challenge
 
-templates:
+actions:
```

Comment:
why had this section been added ?

Reply:
trying to figure out why it shows up that way - local diff only shows templates -> responses

Reply:
could just be an error on github's rendering side